libs/community/langchain_community/chat_models/deepinfra.py

```diff
@@ -86,8 +90,8 @@ def _convert_dict_to_message(_dict: Mapping[str, Any]) -> BaseMessage:
         # Fix for azure
         # Also OpenAI returns None for tool invocations
         content = _dict.get("content", "") or ""
-        if _dict.get("function_call"):
-            additional_kwargs = {"function_call": dict(_dict["function_call"])}
+        if _dict.get("tool_calls"):
```

Comment:
is function calling still supported? if so, would this change have any impact on users of function calling?